### ISSUE TITLE
Add Support for Cucumber V2

### ISSUE THREAD

**Issue by gkushang** [gkushang/cucumber-html-reporter#73]

**Comment by EugeneSnihovsky** [comment]
+1

Generated `report.json` with `cucumber@2.2.0`
```json
[
  {
    "description": "Show error during sign in with wrong credentials",
    "assertions": [
      {
        "passed": true
      },
      {
        "passed": true
      },
      {
        "passed": true
      },
      {
        "passed": true
      },
      {
        "passed": true
      }
    ],
    "events": [
      {
        "emitterEvent": "testPass",
        "name": "When I go to \"sign in\" page",
        "category": "Sign in page scenario: Show error during sign in with wrong credentials",
        "durationMillis": 1183,
        "sourceLocation": {
          "filePath": "/Volumes/WORKSPACE/projects/bdd-reusable/features/sign-in/sign-in.feature",
          "line": 6,
          "column": 1
        }
      },
      {
        "emitterEvent": "testPass",
        "name": "And I fill \"username\" input field with \"alex.a1.at@agiliumlabs.com\"",
        "category": "Sign in page scenario: Show error during sign in with wrong credentials",
        "durationMillis": 156,
        "sourceLocation": {
          "filePath": "/Volumes/WORKSPACE/projects/bdd-reusable/features/sign-in/sign-in.feature",
          "line": 7,
          "column": 1
        }
      },
      {
        "emitterEvent": "testPass",
        "name": "And I fill \"password\" input field with \"wrongPassword\"",
        "category": "Sign in page scenario: Show error during sign in with wrong credentials",
        "durationMillis": 156,
        "sourceLocation": {
          "filePath": "/Volumes/WORKSPACE/projects/bdd-reusable/features/sign-in/sign-in.feature",
          "line": 8,
          "column": 1
        }
      },
      {
        "emitterEvent": "testPass",
        "name": "And I click \"log in\" button",
        "category": "Sign in page scenario: Show error during sign in with wrong credentials",
        "durationMillis": 52,
        "sourceLocation": {
          "filePath": "/Volumes/WORKSPACE/projects/bdd-reusable/features/sign-in/sign-in.feature",
          "line": 9,
          "column": 1
        }
      },
      {
        "emitterEvent": "testPass",
        "name": "Then I should see \"Wrong credentials\" error notification",
        "category": "Sign in page scenario: Show error during sign in with wrong credentials",
        "durationMillis": 2517,
        "sourceLocation": {
          "filePath": "/Volumes/WORKSPACE/projects/bdd-reusable/features/sign-in/sign-in.feature",
          "line": 10,
          "column": 1
        }
      }
    ],
    "duration": 4064
  }
]
```

**Generated html screenshot**

![report screenshot](https://sc-cdn.scaleengine.net/i/4e19707414f91c90ba27dfeb9334c9781.png)

Maybe I missed some option?

**Comment by EugeneSnihovsky** [comment]
My bad. Sorry. I used the wrong `format` in `cucumberOpts`. When I change
```typescript
config.cucumberOpts.format = [
  'pretty'
];
```
to
```typescript
config.cucumberOpts.format = [
  'json:reports/results.json',
  'pretty'
];
```
New file `results.json` appeared in report folder with right data format. And with it your module works perfectly. Thanks for great tool :)

**Comment by gkushang** [comment]
This will be resolved with https://github.com/gkushang/cucumber-html-reporter/pull/81

**Comment by gkushang** [comment]
published the fix with `cucumber-html-reporter@2.0`. Please try it out, and let us know how it works.
